Examples/Examples/FlyoverExampleView.swift

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+import ArcGISToolkit
+
+struct FlyoverExampleView: View {
+    @State private var scene = Scene(
+        item: PortalItem(
+            portal: .arcGISOnline(connection: .anonymous),
+            id: PortalItem.ID("7558ee942b2547019f66885c44d4f0b1")!
+        )
+    )
+    
+    private let initialCamera = Camera(
+        lookingAt: Point(x: 4.4777, y: 51.9244, spatialReference: .wgs84),
+        distance: 1_000,
+        heading: 40,
+        pitch: 90,
+        roll: 0
+    )
+    
+    var body: some View {
+        FlyoverSceneView(
+            initialCamera: initialCamera,
+            translationFactor: 3_000,
+            clippingDistance: 60_000
```

Comment:
```suggestion
            translationFactor: 3_000
```
Clipping distance should not be required in the initializer as it's not needed for a Flyover workflow. However it may be good to support modifying or setting the clipping distance in a future iteration in case it's needed to support a user's specific workflow.

Reply:
Great, thank you